ci/run_envoy_docker.sh

```diff
@@ -9,13 +9,21 @@ function is_windows() {
   [[ "$(uname -s)" == *NT* ]]
 }
 
+function export_env() {
+    default="$1"
+    alt="$2"
+
+    if [ -z "${!default}" ]; then
+        export "$default"="$alt"
```

Comment:
how is this any different to just exporting the vars as is removed below?

Reply:
ok i see it only sets it if its not set already

i think we want to make this more explicit - this is not easy to understand what its doing